### PR TITLE
Update usage-life-cycle.md

### PR DESCRIPTION
**Pull request by pavlenex**
Co-Authored-By: @sqctoucan

This is the part of #202 
The goal of this PR is to simplify the content without altering the context.

### CODE REVIEW

File: guide/designing-products/usage-life-cycle.md [hunk 1]
```diff
@@ -157,9 +160,9 @@ These are expert users with high expectations. They will express strong wishes,
    layout = "float-right-desktop"
 %}
 
-Some users disappear at every step in the life cycle. If you understand where and why they drop off, you have a better chance at fixing those issues and hopefully getting more and more passionate, satisfied users.
+Expects to lose users at every step in the life cycle. This is unavoidable. If you understand where and why they drop off, you can at least reduce the likelihood of it happening by solving for the issues that cause dropoff.
 
-There are various [user research]({{ '/guide/designing-products/user-research/' | relative_url }}) techniques that can help identify these types of issues.
+Barious [user research]({{ '/guide/designing-products/user-research/' | relative_url }}) techniques that can help identify and solve these types of issues
```
Comment (johnsBeharry) [inline guide/designing-products/usage-life-cycle.md:165]:
```suggestion
Various [user research]({{ '/guide/designing-products/user-research/' | relative_url }}) techniques that can help identify and solve these types of issues
```

Comment (pavlenex) [reply guide/designing-products/usage-life-cycle.md]: rofl

File: guide/designing-products/usage-life-cycle.md [hunk 2]
```diff
@@ -157,9 +160,9 @@ These are expert users with high expectations. They will express strong wishes,
    layout = "float-right-desktop"
 %}
 
-Some users disappear at every step in the life cycle. If you understand where and why they drop off, you have a better chance at fixing those issues and hopefully getting more and more passionate, satisfied users.
+Expects to lose users at every step in the life cycle. This is unavoidable. If you understand where and why they drop off, you can at least reduce the likelihood of it happening by solving for the issues that cause dropoff.
 
-There are various [user research]({{ '/guide/designing-products/user-research/' | relative_url }}) techniques that can help identify these types of issues.
+Various [user research]({{ '/guide/designing-products/user-research/' | relative_url }}) techniques that can help identify and solve these types of issues
```
Comment (GBKS) [inline guide/designing-products/usage-life-cycle.md:165]:
```suggestion
There are various [user research]({{ '/guide/designing-products/user-research/' | relative_url }}) techniques that can help identify and solve these types of issues.
```
Sentence had no verb or period.

File: guide/designing-products/usage-life-cycle.md [hunk 3]
```diff
@@ -104,7 +107,7 @@ A user becomes interested in your product. Maybe they have heard about it severa
...
+The user decides to move forward with your product. They are still not familiar with exactly how it works, and their expectations may not match what your product offers. Therefore, it is crucial to tightly manage the [onboarding experience research]({{ '/guide/onboarding/introduction/' | relative_url }}). Your product should make it easy for users to quickly and successfully solve problems while being introduced to your product's basics and best practices. This sets expectations for future interactions. If the first use is frustrating, users are less likely to use your product again—or to recommend it.
```
Comment (GBKS) [inline guide/designing-products/usage-life-cycle.md:110]:
```suggestion
The user decides to move forward with your product. They are still not familiar with exactly how it works, and their expectations may not match what your product offers. Therefore, it is crucial to pay close attention to the design of the [onboarding experience]({{ '/guide/onboarding/introduction/' | relative_url }}). Your product should make it easy for users to quickly and successfully solve problems while being introduced to your product's basics and best practices. This sets expectations for future interactions. If the first use is frustrating, users are less likely to use your product again—or to recommend it.
```
Not sure what is meant by `tightly managing the onboarding experience research`. It's the design of the experience that needs to be well-crafted.

File: guide/designing-products/usage-life-cycle.md [hunk 4]
```diff
@@ -36,9 +36,11 @@ Illustration sources
...
+As we develop new skills, things that are difficult at first eventually become intuitive to us. So when we develop products, it’s a good idea to regularly step back and look at the products we create through users' eyes on that same journey.
 
-The following six sections outline different phases of a typical usage life cycle, from somebody who has never heard of your product to them becoming a power user. It is helpful to think through each phase and consider what the current user needs are, what their understanding of bitcoin and your product likely is, and how you can help them successfully navigate.
+The following six sections outline a typical usage life cycle, from never having heard of your product to becoming one of its power users.
```
Comment (GBKS) [inline guide/designing-products/usage-life-cycle.md:41]:
```suggestion
The following six phases outline a typical usage life cycle, from never having heard of your product to becoming one of its power users.
```
We use sections to refer to parts of the guide, so changing this to phases, which is also used in the following paragraph.

**Review by GBKS — CHANGES_REQUESTED** [review]
Looks good. Just had a few minor nits.

A good future update to this page would be to add more Bitcoin-specific recommendations to each phase.

**Comment by pavlenex** [comment]
@GBKS I committed the suggestions you had, good ones. 🚀 This one is good to go once you take a final look at it.

**Review by GBKS — APPROVED** [review]
Just had another read through and it looks good. I'll open another issue for potentially adding more Bitcoin-specific info to each step.

**Pull request merged**